Neos.Flow/Classes/Http/Helper/ResponseInformationHelper.php

```diff
@@ -30,9 +30,9 @@ abstract class ResponseInformationHelper
      * @throws \InvalidArgumentException
      * @return ResponseInterface
      */
-    public static function createFromRaw(string $rawResponse): ResponseInterface
+    public static function createFromRaw(string $rawResponse, string $responseClassName = Response::class): ResponseInterface
     {
-        $response = new Response();
+        $response = new $responseClassName();
```

Comment:
Shouldn't we check that the result actually is an implementation of `ResponseInterface` here?

Reply:
The 2nd parameter is no longer used (is it?) so we could remove it again

Reply:
Good point.

Reply:
@kitsunet ?